spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.create(name: 'John') }
+
+  it 'title valid with a title' do
+    first_user = User.first
+    post = Post.new(author: first_user, title: 'my first post', commentscounter: 0, likescounter: 0)
+    expect(post).to be_valid
+  end
+
+  it 'has no valid title' do
+    post = Post.new(author: nil, commentscounter: 0, likescounter: 0)
+    expect(post).not_to be_valid
+  end
+
+  it 'has no comment counter' do
+    first_user = User.first
+    post = Post.new(author: first_user, likescounter: 0)
+    expect(post).not_to be_valid
+  end
+
+  it 'updates the postscounter after creation' do
+    Post.create(
+      title: 'Sample Post',
+      commentscounter: 0,
+      likescounter: 0,
+      author: user
+    )
+    user.reload
+    expect(user.postcounter).to eq(1)
+  end
```

Comment:
- [ ] This test case is failing as well, I got the error message screenshot below 👇

<img width="476" alt="Untitled" src="https://github.com/devalibello/blog-app/assets/103386183/e2c2014d-fd6a-48d9-b09b-8b9752f937d3">

Reply:
Hello @OyePriscilla 
I trust you are doing great.
Could you please cross-check the code and tests, everything works fine from my end 😄 .
![image](https://github.com/devalibello/blog-app/assets/114155239/1514db25-9b71-4fab-91a0-a68fe74a8523)

If the error is from my code, I will kindly request you specify the faulty lines of code to me 🐛 

Thanks

Reply:
Ok @devalibello, let's have a zoom call.

Reply:
@devalibello, please kindly take note that a senior colleague also got the same error messages I pointed out. To resolve the issue, please ensure that all the needed parameters are put in to create a user in post_spec.rb file line 4.

<img width="305" alt="Untitled" src="https://github.com/devalibello/blog-app/assets/103386183/75edb04a-8873-4bbb-abb4-f878bfab4ced">

*example* `let(:user)**** { User.create(name: "me", photo: "me", bio: "me", postcounter: 0) }
`
Please don't forget to resolve this failing test as well 👇 

<img width="393" alt="Untitled" src="https://github.com/devalibello/blog-app/assets/103386183/990dcb7c-0de5-4ad0-92a4-4c2887238360">

Reply:
@OyePriscilla Well noted, with thanks.
Thanks for your detailed feedback. I am available now if a call is okay with you 😄. There is always something to learn from others 👨‍🎓